### PR TITLE
CAT-1243 Adjust gradle files to make Catroid work with Android Studio 1....

### PR DESCRIPTION
**Pull request by amix73**
...x

### CODE REVIEW

**Comment by amix73** [comment]
https://jenkins.catrob.at/job/Catroid-Build-SourceTests-StaticChecks/966/

**Comment by avihric** [comment]
You have a lot of lint errors:
https://jenkins.catrob.at/job/Catroid-PullRequest-Test-RELOADED/269/console

File: gradle/code_quality_tasks.gradle [hunk 1]
```diff
@@ -28,7 +28,7 @@ task checkstyle(type: Checkstyle) {
 task pmd(type: Pmd) {
     // current plugin has pmd 4.3 as default
```
Comment (SimonStefan) [inline gradle/code_quality_tasks.gradle]: now lines 29-31 can be remmoved

**Comment by SimonStefan** [comment]
The latest version of the Gradle Plugin has several new lint checks. Please fix the easy ones and ignore the following with a TODO to fix them: IconDensities, IconMissingDensityFolder, RelativeOverlap, RtlSymmetry, RtlHardcoded
Somebody should add a JIRA-Ticket for each of them to fix the occurrences and remove the ignore afterwards.

File: build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.12.+'
+        classpath 'com.android.tools.build:gradle:1.0.0'
```
Comment (SimonStefan) [inline build.gradle]: there is already a later version

**Comment by kaufi** [comment]
I'll have a look at the new Lint checks and write according tickets in the evening today

**Comment by kaufi** [comment]
I made a commit ontop of this one, where the latest gradle plugin is used, the compilation error is gone and the lint warnings are down to 0!
unfortunately I don't have access to the repository any more, so I did the changes in a fork - shall I open a new pullrequest (where Christian's commit is also included)?

**Comment by SimonStefan** [comment]
@kaufi Yes, that would be fine.

**Comment by avihric** [comment]
see #1167

**Pull request closed without merging**